src/macaron/slsa_analyzer/build_tool/base_build_tool.py

```diff
@@ -15,6 +15,29 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
+def find_files(path: str, file_name: str) -> list:
+    """Return a list of file paths that contain a certain file.
+
+    This method searches in the directory recursively.
+
+    Parameters
+    ----------
+    path : str
+        The path to search for the file.
+    file_name : str
+        The name of the file to search.
+
+    Returns
+    -------
+    list:
+        The result list of strings or empty if the file doesn't exist anywhere lower than the path.
+    """
+    pattern = os.path.join(path, "**", file_name)
+    files_detected = glob.glob(pattern, recursive=True)
```

Comment:
I'm not sure if adding this wrapper function over `glob.glob` makes too much sense. All it's doing is calling `glob.glob()` really and I think this could just be done directly at `is_detected` function.

Reply:
I agree. I've removed this function and put the functionality inside `is_detected` instead in [f7bec7f](https://github.com/oracle-samples/macaron/pull/67/commits/f7bec7f96288f729de6b80f58e604a65549cea96)